daemon/mgr/container.go

```diff
@@ -1816,19 +1813,65 @@ func (mgr *ContainerManager) execExitedAndRelease(id string, m *ctrd.Message) er
 	execConfig.Running = false
 	execConfig.Error = m.RawError()
 
-	if io := mgr.IOs.Get(id); io != nil {
-		if err := m.RawError(); err != nil {
-			fmt.Fprintf(io.Stdout, "%v\n", err)
-		}
+	io := mgr.IOs.Get(id)
```

Comment:
change the codes to `releaseContainerIOs` ?

Reply:
I do not think we could do such change. since in the middle of the function, there is a `fmt.Fprintf(io.Stdout, "%v\n", err)`